src/Console/MakeTransformerCommand.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+namespace Rexlabs\Laravel\Smokescreen\Console;
+
+use Illuminate\Console\GeneratorCommand;
+
+class MakeTransformerCommand extends GeneratorCommand
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'make:transformer {model : The namespaced model to transform. e.g. "App\User"}';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Create a new transformer class';
+
+    /**
+     * The type of class being generated.
+     *
+     * @var string
+     */
+    protected $type = 'Transformer';
+
+    /**
+     * Get the stub file for the generator.
+     *
+     * @return string
+     */
+    protected function getStub()
+    {
+        return __DIR__ . '/transformer.stub';
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return bool|null
+     */
+    public function handle()
+    {
+        if (!class_exists($name = $this->argument('model'))) {
+            $confirmed = $this->confirm(
+                'The model to transform does not exist, do you want to generate it?'
+            );
+
+            if ($confirmed) {
+                $this->call('make:model', compact('name'));
```

Comment:
It might be better to not create the model automatically. 
Because: 
a.) I think it will usually be a mistake if you are creating a transformer for a model that doesn't exist
b.) You might want to pass additional options to the command.

I would suggest changing it to something like:
```
That model does not exist, please create one first.  To create it with a migration run :
php artisan make:model -m App\Models\SomeModel
```

Reply:
Great suggestions! I was already planning to use Blade for the stub to ease the reflection but actually didn't think of the extra benefit of publishing the template 👍 

Agreed with not creating models if missing. Probably I'd just limit the error message to:
```
The specified model does not exist, please create one first.
```